test/functional/test_framework/script_util.py

```diff
@@ -128,6 +131,16 @@ def script_to_p2sh_p2wsh_script(script):
     p2shscript = CScript([OP_0, sha256(script)])
     return script_to_p2sh_script(p2shscript)
 
+def bulk_vout(tx, target_vsize):
+    if target_vsize < tx.get_vsize():
+        raise RuntimeError(f"target_vsize {target_vsize} is less than transaction virtual size {tx.get_vsize()}")
+    # determine number of needed padding bytes
+    dummy_vbytes = target_vsize - tx.get_vsize()
+    # compensate for the increase of the compact-size encoded script length
+    # (note that the length encoding of the unpadded output script needs one byte)
+    dummy_vbytes -= len(ser_compact_size(dummy_vbytes)) - 1
+    tx.vout[-1].scriptPubKey = CScript([OP_RETURN] + [OP_1] * dummy_vbytes)
```

Comment:
In 633aa947d684a0fdd3c13353ddd8418cb0195aa5 "test: extract `bulk_vout` from `bulk_tx` so it can be used by wallet tests"

Fine to extract it but since it particularly bulks the last `vout`, we can call the function `bulk_last_vout`? An alternative could be that this function can append the op_return vout in the transaction instead of assuming the caller does it, in which case this function would become `bulk_tx` - but this could be a larger change and can be avoided in this PR.